### PR TITLE
fix groutine leak

### PR DESCRIPTION
**Pull request by mattn**
Fixes https://github.com/fiatjaf/relayer/issues/84

ticker.C does not fire anything after ticker.Stop. So the goroutine will be leak.

### CODE REVIEW

File: handlers.go [hunk 1]
```diff
@@ -54,6 +54,8 @@ func (s *Server) HandleWebsocket(w http.ResponseWriter, r *http.Request) {
 	defer s.clientsMu.Unlock()
 	s.clients[conn] = struct{}{}
 	ticker := time.NewTicker(pingPeriod)
+	stop := make(chan struct{})
+	defer close(stop)
```
Comment (mattn) [inline handlers.go:58]:
```suggestion
```

**Comment by mattn** [comment]
please wait merge. I'm checking this now.

**Comment by mattn** [comment]
Confirmed fixes 
image

**Comment by fiatjaf** [comment]
Thank you very much, sir.

**Pull request merged**